website/docs/docs/billing.md

```diff
@@ -3,14 +3,23 @@ id: billing
 title: Billing and Subscription Management
 ---
 
+:::info
+PactFlow is in the process of migrating customers to the SmartBear Admin system. To determine if you are on the previous system and to find the applicable documentation, consult the following table.
+:::
+
+| SmartBear Admin | PactFlow Legacy |
+|:---:|:--------------:|
+| <img border="1" width="250" src="/ui/plan-widget.png" description="SmartBear Admin Plan Widget" />  | - |
+| If you see this item in your top navigation bar, [SmartBear Admin](https://support.smartbear.com/administration/docs/en/administration.html) is enabled on your account. To Administer your account, consult the SmartBear Admin [docs](https://support.smartbear.com/administration/docs/en/administration.html). | You are on PactFlow's legacy subscription management, and this guide applies. |
```

Comment:
I'd change "To Administer your account, consult the SmartBear Admin (...)" to "To manage your account, refer to the (...)".

Reply:
```suggestion
| If you see this item in your top navigation bar, [SmartBear Admin](https://support.smartbear.com/administration/docs/en/administration.html) is enabled on your account. To manage your account, refer to the SmartBear Admin [docs](https://support.smartbear.com/administration/docs/en/administration.html). | You are on PactFlow's legacy subscription management, and this guide applies. |
```

Reply:
Thanks @MateuszKupczakSB . FYI you can use this "Add a suggestion" button to inline suggest a change (and I can just click "commit suggestion" to accept it:

<img width="298" alt="Screenshot 2024-12-11 at 5 10 21 PM" src="https://github.com/user-attachments/assets/755a81ac-32b7-42d1-9a6d-4ffd2afb995b">

See my comment above, where I've done it on your behalf - means I can just accept and merge from this UI directly!